pkg/test/resourcefixture/testdata/basic/logging/v1beta1/logginglogexclusion/billingexclusion/update.yaml

```diff
@@ -15,8 +15,14 @@
 apiVersion: logging.cnrm.cloud.google.com/v1beta1
 kind: LoggingLogExclusion
 metadata:
+  annotations:
+    cnrm.cloud.google.com/deletion-policy: "abandon"
```

Comment:
Could you help me understand why `abandon` is needed for LoggingLogExclusion?

Reply:
I copy-pasted to a wrong place. We should not make the changes to this test. I will revert it.